deepmd/nvnmd/entrypoints/mapt.py

```diff
@@ -252,7 +436,7 @@ def build_s2G(self, s):
                 xyz_scatter = s
                 for ll in range(1, len(outputs_size)):
                     w, b = get_filter_weight(nvnmd_cfg.weight, tt, tt2, ll)
-                    w, b = np.float32(w), np.float32(b)
+                    w, b = np.float64(w), np.float64(b)
```

Comment:
It is meaningless to cast data with low precision to that with high precision

Reply:
All operations in this file that set the value to float64 are intended to avoid excessive computation error accumulation during GPU testing.